planner/core/rule_partition_processor.go

```diff
@@ -47,15 +47,17 @@ const FullRange = -1
 // Used by static partition prune mode.
 //
 // create table t (id int) partition by range (id)
-//   (partition p1 values less than (10),
-//    partition p2 values less than (20),
-//    partition p3 values less than (30))
+//
```

Comment:
This is weird, please remove the indent so that we don't have to insert new empty line.